examples/tutorials/Protein_Generation.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>Might be worth adding some commentary</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/4423/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
<p>Added commentary about the generated protein. Also pointed out the straight lines are due to 2D projection of the 3D alpha helices.</p>